packages/components/src/palette-edit/index.tsx

```diff
@@ -261,27 +261,30 @@ function Option< T extends Color | Gradient >( {
 	);
 }
 
-function isTemporaryElement(
+/**
+ * Checks if a color or gradient is a temporary element by testing against default values.
+ */
+export function isTemporaryElement(
 	slugPrefix: string,
 	{ slug, color, gradient }: Color | Gradient
-) {
+): Boolean {
```

Comment:
Minor comment, but [we should use `boolean` instead of `Boolean`](https://stackoverflow.com/questions/64443288/what-is-the-difference-between-boolean-and-boolean-in-typescript)

Reply:
Oh! Thanks. I'll fix in an upcoming, related PR 👍🏻

Reply:
Great catch, thanks Marco!